packages/rpc/src/configuration.ts

```diff
@@ -0,0 +1,8 @@
+export namespace ConfigurationRpc {
+  export const SetFunctionName = 'configuration.set';
+  export const GetFunctionName = 'configuration.get';
+
+  export type Configuration = {
```

Comment:
One thing we have to watch out for @dustinbyrne is that the backend is going to **require** the Configuration to be set before any of the methods will have a sane behavior. Otherwise everything is going to return empty results.

Reply:
So the RPC process should be launched as early as possible as the AppMap extension is initializing, and the RPC configuration must be kept up to date. For example, updated when appmap.yml add/remove events are handled, and we may want to back that up with a polling process that also updates the config every 10 seconds or so. Just to make sure we don't miss anything. Events are nice when they work, but they aren't 100% reliable so it's important to have a polling backup.